packages/common/src/ui/layout/tables/components/index.tsx

```diff
@@ -15,16 +14,14 @@ export const BasicCell = <T extends DomainObject>({
   const d = useFormatDate();
 
   return (
-    <span
+    <div
```

Comment:
offroad to fix this:

<img width="345" alt="Screen Shot 2021-12-01 at 5 56 46 PM" src="https://user-images.githubusercontent.com/9192912/144342580-9d52e13b-633a-4c3a-896a-063d5fc14aa6.png">